code/modules/cargo/packs.dm

```diff
@@ -1016,6 +1024,55 @@
 /datum/supply_pack/materials
 	group = "Canisters & Materials"
 
+/datum/supply_pack/materials/diamond20
+	name = "20 Diamond Sheets"
+	desc = "Create with 20 sheets of diamonds. Usefull for M.R.S.P.A.C.M.A.N.-type portable generators."
+	cost = 15000 // 15000 worth of diamonds
```

Comment:
This is either too high an amount of diamonds or too cheap of a crate. You can decide which. My opinion is it's too many diamonds **and** too cheap.